index.js

```diff
@@ -0,0 +1,146 @@
+const bioBody = document.querySelector('#bio-body')
+const portfolioBody = document.querySelector("#portfolio-body");
+const recycleBody = document.querySelector("#recycle-body");
+const contactBody = document.querySelector("#contact-body");
+const poemBody = document.querySelector("#poem-body");
+
+const sections = [
+    {title:"About Me",body:bioBody,color:'blue',ext:'doc'},
+    {title:"Portfolio",pinned:true, body:portfolioBody,color:'green',ext:'xls'},
+    {title:"Contact", body:contactBody,color:'blue',ext:'json'},
+    {title:"Chaucer's Fortune", body:poemBody},
+    {},{},{},{},
+    {title:'Recycle',body:recycleBody,ext:'recycle'},
+    {},{},{},{},{},{},{},{},{},
```

Comment:
nit: unless these are necessary for the functionality, consider removing these empty objects from the array of sections